### PR TITLE
Fix #110: Documentation page on PyPI has broken formatting

### LINKED ISSUE #110: Documentation page on PyPI has broken formatting
[Documentation page](https://pypi.python.org/pypi/tenacity) on PyPI has broken formatting. 

The beginning looks like this:
```
Retry code until it succeeeds
Tenacity
========
.. image:: https://img.shields.io/pypi/v/tenacity.svg
:target: https://pypi.python.org/pypi/tenacity

.. image:: https://img.shields.io/travis/jd/tenacity.svg
:target: https://travis-ci.org/jd/tenacity

.. image:: https://img.shields.io/badge/SayThanks.io-%E2%98%BC-1EAEDB.svg
:target: https://saythanks.io/to/jd

Tenacity is an Apache 2.0 licensed general-purpose retrying library, written in
Python, to simplify the task of adding retry behavior to just about anything.
It originates from `a fork of retrying
<https: github.com="" rholder="" retrying="" issues="" 65="">`_.

The simplest use case is retrying a flaky function whenever an `Exception`
occurs until a value is returned.

.. testcode::

import random
from tenacity import retry

@retry
def do_something_unreliable():
if random.randint(0, 10) > 1:
raise IOError("Broken sauce, everything is hosed!!!111one")
else:
return "Awesome sauce!"

print(do_something_unreliable())

.. testoutput::
:hide:

Awesome sauce!

Features
--------
```

### PR DESCRIPTION
**Pull request by penguinolog**
use parseable `.. code-block:: python` and `.. ::` for output. Validate readme on CI.

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -30,6 +30,11 @@ deps = flake8
        flake8-logging-format
 commands = flake8
 
+[testenv:readme]
+deps = 
+    collective.checkdocs
```
Comment (jd) [inline tox.ini]: How different it is than `doc8`?

File: doc/source/index.rst [hunk 2]
```diff
@@ -17,7 +17,7 @@ It originates from `a fork of retrying
 The simplest use case is retrying a flaky function whenever an `Exception`
 occurs until a value is returned.
 
-.. testcode::
+.. code-block:: python
```
Comment (jd) [inline doc/source/index.rst]: sounds to me that it won't use doctest anymore if we change that, no?

Comment (penguinolog) [reply doc/source/index.rst]: Yep. For doctests specific tags present, but I don't remember for usage possibility for readme.

File: tox.ini (hunk 1 above)
Comment (penguinolog) [reply tox.ini]: It just validate for possibility to render Readme on PyPI side. Nothing more.

File: doc/source/index.rst (hunk 2 above)
Comment (jd) [reply doc/source/index.rst]: So that's not gonna fly. Best thing is to just replace the README.rst by a link to the doc at this stage.

Comment (penguinolog) [reply doc/source/index.rst]: Ok, closing PR

**Pull request closed without merging**